IaC/PnP/monitoring/aks/online/aks_secure_baseline/configuration/networking/firewall_application_rule_collection_definition.tfvars

```diff
@@ -0,0 +1,135 @@
+azurerm_firewall_application_rule_collection_definition = {
+  aks = {
+    name     = "aks"
+    action   = "Allow"
+    priority = 100
+    ruleset = {
+      aks = {
+        name = "aks"
+        # source_addresses = [
+        #   "*",
+        # ]
+        source_ip_groups_keys = [
+          "aks_ip_group1"
+        ]
+        fqdn_tags = [
+          "AzureKubernetesService",
+        ]
+      },
+    }
+  }
+  packages = {
+    name     = "packages"
+    action   = "Allow"
+    priority = 110
+    ruleset = {
+      ubuntu = {
+        name = "ubuntu"
+        # source_addresses = [
+        #   "*",
+        # ]
+        source_ip_groups_keys = [
+          "aks_ip_group1"
+        ]
+        target_fqdns = [
+          "security.ubuntu.com",
+          "azure.archive.ubuntu.com",
+          "archive.ubuntu.com",
+          "changelogs.ubuntu.com",
+        ]
+        protocol = {
+          https = {
+            port = "443"
+            type = "Https"
+          }
+          http = {
+            port = "80"
+            type = "Http"
+          }
+        }
+      },
+      docker = {
+        name = "docker"
+        # source_addresses = [
+        #   "*",
+        # ]
+        source_ip_groups_keys = [
+          "aks_ip_group1"
+        ]
+        target_fqdns = [
+          "download.docker.com", # Docker
+          "*.docker.io",         # Docker images
+          "*.docker.com"         # Docker registry
+        ]
+        protocol = {
+          http = {
+            port = "443"
+            type = "Https"
+          }
+        }
+      },
+      tools = {
+        name = "tools"
+        # source_addresses = [
+        #   "*",
+        # ]
+        source_ip_groups_keys = [
+          "aks_ip_group1"
+        ]
+        target_fqdns = [
+          "packages.microsoft.com",
+          "azurecliprod.blob.core.windows.net", # Azure cli
+          "packages.cloud.google.com",          # kubectl
+          "apt.kubernetes.io",                  # Ubuntu packages for kubectl
+          "*.snapcraft.io",                     # snap to install kubectl
+        ]
+        protocol = {
+          http = {
+            port = "443"
+            type = "Https"
+          }
+        }
+      },
+      github = {
+        name = "github"
+        # source_addresses = [
+        #   "*",
+        # ]
+        source_ip_groups_keys = [
+          "aks_ip_group1"
+        ]
+        target_fqdns = [
+          "api.github.com",
+          "github.com",
+          "github-production-release-asset-2e65be.s3.amazonaws.com",
+        ]
+        protocol = {
+          http = {
+            port = "443"
+            type = "Https"
+          }
+        }
+      },
+      mcr = {
```

Comment:
@gortegaMS Added a needed section here for creating a firewall rule for github container registry: https://github.com/retaildevcrews/caf-terraform-landingzones-starter/blob/928520972e4493459739fe882d5dadc91a2e196e/enterprise_scale/construction_sets/aks/online/aks_secure_baseline/configuration/networking/firewall_application_rule_collection_definition.tfvars#L113

Reply:
not needed for this environment right now.  once we get to the loderunner workload we should be using the baseline from the Deploy team.